.env.sample

```diff
@@ -1,2 +1,2 @@
-BASE_PATH = /google-map-react-examples/
+REACT_APP_BASE_PATH = /
```

Comment:
Why this change? We need this base path to make it work in github.io

Reply:
To make it work locally - I had an issue where CRA couldn't see the var unless it was prefixed with `REACT_APP_ `

The change of the path was to have the homepage fire up once you run `yarn start`

Reply:
@callamwilliams Sounds good, but we might need to keep it as `/google-map-react-examples` in order to make it work live here:
- https://google-map-react.github.io/google-map-react-examples/

Reply:
OK, updated - removed `.env.sample` to keep it clean

Reply:
@callamwilliams Looks great, did you check in your fork that the `gh-pages` work? The info is in the readme. I think you might need to set up the environment to prod in the `deploy` method of `package.json` so it knows which `.env` file to take. Does this makes sense?

Reply:
Had to adjust the paths based on environments a little, but working correctly now - GH pages sets process.env to production automatically when pushed which is nice.

<img width="422" alt="screenshot 2018-11-03 at 16 31 26" src="https://user-images.githubusercontent.com/1899003/47954760-f80c4c00-df85-11e8-9bc9-90f939dd0f07.png">

Reply:
Thanks @callamwilliams, great work really, I just approved, waiting for circle-ci to do the checks in order to merge.